pages/index.page.tsx

```diff
@@ -273,22 +280,38 @@ const Home = (props: any) => {
               <h3 className='text-white text-xl mb-4'>Used by</h3>
 
               <div className='grid md:grid-cols-2 lg:grid-cols-4 gap-6 mx-auto items-center w-1/3 md:w-100 text-center'>
-                <img
-                  src='/img/logos/usedby/zapier-logo_white.png'
-                  className='w-40 mx-auto'
-                />
-                <img
-                  src='/img/logos/usedby/microsoft-white.png'
-                  className='w-40 mx-auto'
-                />
-                <img
-                  src='/img/logos/usedby/postman-white.png'
-                  className='w-40 mx-auto'
-                />
-                <img
-                  src='/img/logos/usedby/github-white.png'
-                  className='w-40 mx-auto'
-                />
+                {isClient && (
```

Comment:
Please remove the conditional rendering of the image and add meaningful alt texts